.jenkins/actions/fetch_caches.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
```

Comment:
Unless we call this directly using `jenkins.sh` (I don't think we do?) this should be moved up to `.jenkins`.